docs/deployment-process/lifecycles/index.md

```diff
@@ -41,6 +41,16 @@ At this point, you can add one or multiple environments, or leave the default **
 
 If you have a project setup with [Automatic Release Creation](/docs/deployment-process/releases/automatic-release-creation.md) and set your first phase and environment to automatically deploy, pushing a package to the internal library will trigger both a release, and a deployment to that environment.
 
+:::hint
+**Tenants and automatic-environments**
+
+1. If tenanted deployments are allowed for the project, we will attempt to enqueue a new deployment for each tenant connected to the automatic-environment(s), taking the following into consideration
```

Comment:
As each point is a complete sentence, I'd add punctuation at the end of each line. A colon : for his one and full stops for the others.